<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AParticleAnalyzerWidget</class>
 <widget class="QWidget" name="AParticleAnalyzerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Energy distribution histogram</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>    Bins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbEnergyBins">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEnergyFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEnergyTo"/>
     </item>
     <item>
      <widget class="QComboBox" name="cobEnergyUnits">
       <item>
        <property name="text">
         <string>MeV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>keV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>eV</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbTimeWindow">
       <property name="text">
        <string>Time window from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTimeWindowFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTimeWindowTo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ns</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbStopTracking">
       <property name="text">
        <string>Stop particle tracking</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSingleInstance">
       <property name="text">
        <string>Single instance for all copies</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
